jdbc/postgres/src/test/resources/conscrypt-security.properties

```diff
@@ -0,0 +1,13 @@
+security.provider.1=Conscrypt
+security.provider.2=SUN
```

Comment:
They should all be here, but Conscrypt should come first so it gets loaded first.

Reply:
This file fully replaces the built-in JVM security properties. So we need to set all of the default security properties, not just the few properties that are relevant to prioritizing the Conscrypt JCE library.

Reply:
Could we say that in a comment in the file? Future maintainers are going to struggle to understand why this is here otherwise. Also, do we need all the extra whitespace below?

Reply:
I added some comments to the file to clarify.